### PR TITLE
Add 'DB Regio AG' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.datenanfragen.de/#!doc=%7B%22slug%22%3A%22dbregio%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22categories%22%3A%5B%22travel%22%5D%2C%22name%22%3A%22DB%20Regio%20AG%22%2C%22runs%22%3A%5B%22bahn.de%22%5D%2C%22address%22%3A%22Europa-Allee%2070-76%5Cn60486%20Frankfurt%20am%20Main%22%2C%22phone%22%3A%22030%202970%22%2C%22email%22%3A%22datenschutz.regio%40deutschebahn.com%22%2C%22web%22%3A%22https%3A%2F%2Fwww.dbregio.de%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.dbregio.de%2Fhome%2Fdatenschutz%22%2C%22https%3A%2F%2Fwww.dbregio.de%2Fimpressum%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1292%22%5D%2C%22suggested-transport-medium%22%3A%22email%22%2C%22comments%22%3A%5B%22Das%20Portal%20bahn.de%20wird%20von%20DB%20Vertrieb%20GmbH%2C%20DB%20Fernverkehr%20AG%20und%20DB%20Regio%20AG%20als%20gemeinsam%20Verantwortliche%20gem.%2028%20DSGVO%20betrieben.%22%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/dbregio.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "DB Regio AG",
+    "runs": [
+        "bahn.de"
+    ],
+    "address": "Europa-Allee 70-76\n60486 Frankfurt am Main",
```
Comment (moriarty-jim) [inline companies/dbregio.json:13]:
```suggestion
    "address": "Europa-Allee 70-76\n60486 Frankfurt am Main\nDeutschland",
```
fixed address format

File: companies/dbregio.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "slug": "dbregio",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "DB Regio AG",
+    "runs": [
+        "bahn.de"
```
Comment (moriarty-jim) [inline companies/dbregio.json:11]: [https://www.bahn.de/datenschutz](https://www.bahn.de/datenschutz)
-> "Wer ist verantwortlich für die Datenerhebung und -verarbeitung?" lists multiple companies responsible for bahn.de.
-> "Welche Rechte haben Nutzerinnen und Nutzer von bahn.de/bahn.com?" states that DB Vertrieb GmbH should be contacted. 
What should be done here?

File: companies/dbregio.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    ],
+    "address": "Europa-Allee 70-76\n60486 Frankfurt am Main",
+    "phone": "030 2970",
```
Comment (moriarty-jim) [inline companies/dbregio.json:14]:
```suggestion
    "phone": "+49 30 2970",
```
fixed phone format

File: companies/dbregio.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    "email": "datenschutz.regio@deutschebahn.com",
+    "web": "https://www.dbregio.de",
+    "sources": [
+        "https://www.dbregio.de/home/datenschutz",
+        "https://www.dbregio.de/impressum",
+        "https://github.com/datenanfragen/data/pull/1292"
+    ],
+    "suggested-transport-medium": "email",
+    "comments": [
+        "Das Portal bahn.de wird von DB Vertrieb GmbH, DB Fernverkehr AG und DB Regio AG als gemeinsam Verantwortliche gem. 28 DSGVO betrieben."
```
Comment (moriarty-jim) [inline companies/dbregio.json:24]: see above (line 11)

**Review by moriarty-jim — CHANGES_REQUESTED** [review]
fixed phone, address format; responsibility for bahn.de unclear & should be discussed

File: companies/dbregio.json (hunk 2 above)
Comment (baltpeter) [reply companies/dbregio.json:11]: Hm, we currently have a record for DB Vertrieb GmbH (https://www.datarequests.org/company/deutsche-bahn) that lists all those companies and bahn.de under `runs`.

According to "Welche Daten verarbeiten wir und wie und warum verarbeiten wir Ihre Daten?", DB Vertrieb GmbH is involved in all processing activities. And as you said, the privacy also tells you to contact them and even says "Sofern Sie uns anschreiben, Ihr Anliegen aber den Verantwortungsbereich des von Ihnen genutzten Dienstes betrifft, leiten wir dieses an ihn weiter."

I can also confirm from a prior request to DB Vertrieb GmbH, that the reply includes the data on everything.

So, I think we don't need this record. But we should update the DB Vertrieb GmbH one as at least the address of that one is outdated. What do you think?

Comment (moriarty-jim) [reply companies/dbregio.json:11]: Agreed. Both options aren't perfect, but if a request to DB Vertrieb GmbH reports back data from all related companies, this is IMHO definitely the better option.

**Comment by baltpeter** [comment]
Closed as per: https://github.com/datenanfragen/data/pull/1292#discussion_r720652830

File: companies/dbregio.json (hunk 2 above)
Comment (baltpeter) [reply companies/dbregio.json:11]: I have updated the contact details in b81fa4facb5e7b176bc1c1e4c2fbade414de5091.

**Pull request closed without merging**